XcodeServerSDK/Utils/JSON.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol JSONSerializable {
-    init?(json: NSDictionary)
+    init?(json: NSDictionary) throws
```

Comment:
Probably yes and if by any luck no we should change those initializers ASAP. This issue is mentioned in #29 and IMHO it's a **blocker** ⛔️